_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,405 @@
+                <option value="">- Choose one -</option>
+                <option value="Red">Red</option>
+                <option value="Orange">Orange</option>
+                <option value="Yellow">Yellow</option>
+                <option value="Green">Green</option>
+                <option value="Blue">Blue</option>
+                <option value="Indigo">Indigo</option>
+                <option value="Violet">Violet</option>
+            </select>
+        </div>
+
+        <div class="field">
+            <label class="form-label" for="select_02">Enhanced select menu</label>
+            <select class="form-input input-select replace-select" name="select_02" id="select_02">
+                <option value="">- Choose one -</option>
+                <option value="Red">Red</option>
+                <option value="Orange">Orange</option>
+                <option value="Yellow">Yellow</option>
+                <option value="Green">Green</option>
+                <option value="Blue">Blue</option>
+                <option value="Indigo">Indigo</option>
+                <option value="Violet">Violet</option>
+            </select>
+        </div>
+        <div class="field">
+            <label class="form-label" for="select_02_err">Enhanced select menu with error</label>
+            <select class="form-input input-select replace-select has-error" name="select_02_err" id="select_02_err">
+                <option value="">- Choose one -</option>
+                <option value="Red">Red</option>
+                <option value="Orange">Orange</option>
+                <option value="Yellow">Yellow</option>
+                <option value="Green">Green</option>
+                <option value="Blue">Blue</option>
+                <option value="Indigo">Indigo</option>
+                <option value="Violet">Violet</option>
+            </select>
+        </div>
+        <div class="field">
+            <label class="form-label" for="select_02_suc">Enhanced select menu with success</label>
+            <select class="form-input input-select replace-select has-success" name="select_02_suc" id="select_02_suc">
+                <option value="">- Choose one -</option>
+                <option value="Red">Red</option>
+                <option value="Orange">Orange</option>
+                <option value="Yellow">Yellow</option>
+                <option value="Green">Green</option>
+                <option value="Blue">Blue</option>
+                <option value="Indigo">Indigo</option>
+                <option value="Violet">Violet</option>
+            </select>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6 example-set-hd">Radios</h4>
+<div class="example-set">
+    <fieldset class="field-group form-group">
+        <legend class="field-group-hd form-label">Default radio buttons</legend>
+        <div class="field">
+            <label class="form-label label-inline" for="updates_mobile_text_01">
```

Comment:
Can/should we always count on a ``<label>`` element wrapping our checkbox/radio type inputs? Why not keep these separate and use the ``for`` and ``id`` attributes to associate these for more flexible markup?